src/openfermion/ops/_fermion_operator.py

```diff
@@ -226,131 +202,11 @@ class FermionOperator(object):
     3.17 2^ 1 - 66.2 * 8^ 7 6^ 2
     The Fermion Operator class overloads operations for manipulation of
     these objects by the user.
-
-    Attributes:
```

Comment:
Are you completely getting rid of the attributes docs? Or is that part of the docs going to be inherited or something?

Reply:
All the attributes are inherited. I'm not sure what our convention about documenting subclasses is (if we have one) but now that I think of it, it would be good to document exactly what `actions` and `action_strings` are for subclasses of SymbolicOperator.

Reply:
These are the most important classes in the whole library. I think it is very likely that many people actually take a look at these docs. So I think we should error on the side of having excessive documentation here.